README.md

```diff
@@ -6,13 +6,16 @@
 
 ## Fail2Ban: ban hosts that cause multiple authentication errors
 
-Fail2Ban scans log files like /var/log/pwdfail and bans IP that makes too many
-password failures. It updates firewall rules to reject the IP address. These
-rules can be defined by the user. Fail2Ban can read multiple log files such as
-sshd or Apache web server ones.
-
-Fail2Ban is able to reduce the rate of incorrect authentications attempts
-however it cannot eliminate the risk that weak authentication presents.
+Fail2Ban scans log files like `/var/log/auth.log` and bans IP addresses
+having too many failed login attempts.
+It does this by updating system firewall rules to reject new connections
+from those IP addresses, for a configurable amount of time.
+Fail2Ban comes out-of-the-box ready to read many standard log files, such as those
+for sshd and Apache, and is easy to configure to read any log file you choose, for
+any error you choose.
```

Comment:
I would prefer "filled" paragraph to the width consistent with the rest (around 78?)